captslog/ui/main.py

```diff
@@ -1,5 +1,5 @@
 from __future__ import absolute_import, unicode_literals
-from PyQt4 import QtCore, QtGui
+from PySide import QtCore, QtGui
```

Comment:
I'll merge this after we are done with presentation, but for the presentation, keep your branch alive.